openupgradelib/openupgrade.py

```diff
@@ -618,10 +618,15 @@ def rename_fields(env, field_spec, no_deep=False):
         # Rename appearances on export profiles
         # TODO: Rename when the field is part of a submodel (ex. m2one.field)
         cr.execute("""
-            UPDATE ir_exports_line
+            UPDATE ir_exports_line iel
             SET name = %s
-            WHERE name = %s
-            """, (old_field, new_field),
+            FROM ir_exports ie,
+                ir_model im
+            WHERE iel.name = %s
+                AND ie.id = iel.export_id
+                AND im.id = ie.model_id
+                AND im.model = %s
+            """, (old_field, new_field, model),
```

Comment:
```suggestion
            """, (new_field, old_field, model),
```
🤔

Reply:
the same for ir_translation query just above in the code

Reply:
It has been this way all the time? :scream: